### ISSUE TITLE
how can the diagram in readme be generate

### ISSUE THREAD

**Issue by allanGoldZhiKang** [SDWebImage/SDWebImage#2697]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 10 / Xcode 9
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by bpoplauschi** [comment]
As it is now, It can't be auto-generated. I just update it manually.
But if you have any ideas of tools that generate class diagrams from code, feel free to comment back

**Comment by allanGoldZhiKang** [comment]
> As it is now, It can't be auto-generated. I just update it manually.
> But if you have any ideas of tools that generate class diagrams from code, feel free to comment back

Can you tell me which tool or software you use to generate these diagrams
